### PR TITLE
web: remove `.theme-redesign` selector from search

### PR DESCRIPTION
**Pull request by 5h1rU**
## Changes

- Removed `.theme-redesign` usage from search.

Part of https://github.com/sourcegraph/sourcegraph/issues/20847.

### CODE REVIEW

File: client/web/src/search/results/progress/StreamingProgress.scss [hunk 1]
```diff
@@ -9,23 +9,14 @@
...
-            padding-right: 0.5rem;
-            border-radius: var(--border-radius);
-
-            &--in-progress {
-                color: var(--link-color);
-            }
-        }
+        color: var(--body-color);
+        padding-left: 0.5rem;
+        padding-right: 0.5rem;
```
Comment (limitedmage) [inline client/web/src/search/results/progress/StreamingProgress.scss:14]: Missing top/bottom padding here.
```suggestion
        padding: 0.375rem 0.5rem
```

File: client/web/src/search/results/progress/StreamingProgress.scss [hunk 2]
```diff
@@ -36,39 +27,30 @@
     }
 
     &__count-icon {
-        .theme-redesign & {
-            display: none;
-        }
+        display: none;
     }
```
Comment (limitedmage) [inline client/web/src/search/results/progress/StreamingProgress.scss:31]: We should remove the icon in the .tsx file instead.

File: client/web/src/search/results/progress/StreamingProgress.scss [hunk 3]
```diff
@@ -36,39 +27,30 @@
...
 
     &__skipped {
+        flex-grow: 1;
```
Comment (limitedmage) [inline client/web/src/search/results/progress/StreamingProgress.scss:34]: This should be added to `.streaming-progress` itself instead

File: client/web/src/search/results/progress/StreamingProgress.scss [hunk 4]
```diff
@@ -36,39 +27,30 @@
...
+        &--warning {
+            color: var(--infobar-warning-color);
+        }
```
Comment (limitedmage) [inline client/web/src/search/results/progress/StreamingProgress.scss:37]: Why is this added?

**Review by valerybugakov — APPROVED** [review]
